lib/workers/repository/dependency-dashboard.ts

```diff
@@ -624,6 +624,7 @@ export async function ensureDependencyDashboard(
       body: platform.massageMarkdown(issueBody, config.rebaseLabel),
       labels: config.dependencyDashboardLabels,
       confidential: config.confidential,
+      isPinned: true,
```

Comment:
This should be configurable via a new config option, so users can decide for themselves.